Classes/Backend/DynamicFlexForm.php

```diff
@@ -100,7 +100,7 @@ protected function patchTceformsWrapper(array $dataStructure, $parentIndex = NUL
 				$dataStructure[$index] = $this->patchTceformsWrapper($subStructure, $index);
 			}
 		}
-		if (isset($dataStructure['config']) && $parentIndex !== 'TCEforms') {
+		if (isset($dataStructure['config']) && isset($dataStructure['config']['type']) && $parentIndex !== 'TCEforms') {
```

Comment:
The first `isset($dataStructure['config'])` can then be removed since `isset($dataStructure['config']['type'])` will also safely return FALSE if `$dataStructure['config']` does not exist.